server/chat-formatter.ts

```diff
@@ -307,7 +307,7 @@ class TextFormatter {
 						let dir = key;
 						if (key === 'item') dir += 's';
 
-						uri = `//dex.pokemonshowdown.com/${dir}/${toID(term)}`;
+						uri = `//${Config.routes.dex}/${dir}/${toID(term)}`;
```

Comment:
This is used by client, and can't really have external dependencies.

Reply:
Hmm, yes, I see that its cross compiled there. I'll punt on this for now. Reverted